test/unit/simple_layout_test.rb

```diff
@@ -13,7 +13,7 @@ def setup
 
     assert SimpleLayout.new(@provider).import!
     assert_equal 2, @provider.layouts.count
-    assert_equal 14, @provider.builtin_static_pages.count
+    assert_equal 0, @provider.builtin_static_pages.count
```

Comment:
This test doesn't make sense anymore after this change. Why not mocking a static page and ensure it's imported?

Reply:
I've been thinking about this but here's the thing: the test is about `SimpleLayout` importing the forum templates, not testing the method itself. It's not really a unit test but an integration test of sorts. Looking at the other tests they all test against hard data, nothing is mocked.

So we could mock that SimpleLayout has some builtin static templates but I don't think that's what this test is for.